src/site/content/en/blog/prefers-color-scheme/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "prefers-color-scheme: Hello darkness, my old friend"
+title: 'prefers-color-scheme: Hello darkness, my old friend'
 subhead: Overhyped or necessity? Learn everything about dark mode and how to support it to the benefit of your users!
 authors:
   - thomassteiner
```

Comment:
You may want to update the "updated" date as well.

Reply:
Done.